.github/ISSUE_TEMPLATE/BUG_REPORT.md

```diff
@@ -0,0 +1,58 @@
+---
+name: Bug Report
+about: Create a report to help us improve
+labels: bug
+---
+
+<!--
+Thanks for reporting a bug report to CliMa, the Climate Machine!
```

Comment:
```suggestion
Thanks for reporting a bug report to CliMA, the Climate Machine!
```
according to our website, this is the capitalization

Reply:
CliMa = Climate Machine
CliMA = Climate Modeling Alliance

Reply:
I thought CliMA was the Climate Modeling Alliance and CliMa was the Climate Machine?

Reply:
I thought CLIMA was the Climate Machine? (I guess this is why Dan said ca. 15 months ago we should not use lowercase because it will lead to confusion ... :)

Reply:
Thanks for the clarification. I'll update everything to be all caps.